components/layouts/TheMenu/index.vue

```diff
@@ -166,12 +166,105 @@ export default class TheMenu extends Mixins(MenuMixin) {
     padding-bottom: $spacing-05;
     display: flex;
     justify-content: flex-end;
+    align-items: center;
   }
 
   &__navigation-level {
+    padding-top: 0;
+    padding-bottom: 0;
+
     @include mq($until: large) {
       display: none;
     }
+
+    &__list {
+      display: flex;
+      align-items: center;
+
+      // targeting specific links for consistent spacing
+      &-item:last-child .menu__link {
+        margin-right: 0;
+      }
+
+      &-item:nth-child(2) .menu__link {
+        margin-right: $spacing-07;
+      }
+    }
+
+    // menu dropdown overrides
+    .bx--form-item {
```

Comment:
Do you mind to separate into two sections the override of the carbon design components and the styles of the component? You can see an example [here](https://github.com/Qiskit/qiskit.org/blob/master/components/learn/TheLearningResourceList.vue#L191)

Reply:
Okay @y4izus - the overrides are separated into their own `style` block

thank you!